File: Conversión de Bases/Conversión de bases.java
public class Main {

  public static void main(String[] args) {
    long startTime = System.nanoTime();
    int decimal = 173; 
    String cal = "";
    while (decimal > 0) {
      int residuo = decimal % 2;
      cal= String.valueOf(residuo)+cal;
      decimal=decimal/2;
    }
    long endTime = System.nanoTime();

    System.out.println("Su binario es: "+ cal);
    System.out.println("Tiempo de ejecución: " + (endTime-startTime)/1e6 + "ms");
  }
  
}

File: Conversión de Bases/Conversión de bases.py
import time

def DecAbin(decimal):
  bin =[]
  print ("\nNumero en decimal: " , decimal)
  
  while decimal != 0:
    residuo=decimal%2 
    bin.append(residuo)
    decimal=(decimal//2)
    bin.reverse()
  print("\nEste es su binario:\n",bin)

decimal=int(input("Ingrese el decimal que desea convertir a binario:"))

inicio = time.time()

DecAbin(decimal)

fin = time.time()
print("\nEl tiempo de ejecución fue de:\n ",fin-inicio)